business_requirement_earned_value/report/br_earned_value_report.py

```diff
@@ -94,6 +94,8 @@ def _select(self):
                                    project_task pt
                                WHERE
                                    pt.business_requirement_id = br.id
+                                   AND
+                                   SUM(res.unit_price) > 0
```

Comment:
Doing so we would exclude any line with price = 0.
I am not sure I want to do this (The sum of quantity would be removed as well)